migration/v0.9.x-v0.10.x/upgrade-apps.md

```diff
@@ -0,0 +1,36 @@
+
+# Upgrade v0.9.x to v0.10.0
+
+You will need to follow these steps in order to upgrade each Compliant Kubernetes environment from v0.8.x to v0.9.0. The instructions assume that you work from the root of the `compliantkubernetes-apps` repository, has the latest changes already pulled and configured `CK8S_CONFIG_PATH` correctly.
+
+1. Checkout the new release: `git checkout v0.10.0`.
+
+2. Run init to get new defaults: `./bin/ck8s init`
+
+3. Upgrade workload cluster applications
+  ```bash
+  ./bin/ck8s apply wc
+  ```
+
+5. Upgrade service cluster applications
+  ```bash
+
+   # Upgrade Elasticsearch
+   ./bin/ck8s ops helmfile sc -l app=opendistro apply
+
+   # Wait for master pod 0 to be up and for elasticsearch to have started
+
+   # Reload security config
```

Comment:
You need to reload the config in a new terminal right? If so then write that here, I know that I have missed that in the past and waited until the release timed out.

Reply:
I've added a note.

Reply:
Thanks :slightly_smiling_face: